bin/common/versions.sh

```diff
@@ -24,7 +24,7 @@ export STAMPY_MAJOR=$(echo "$STAMPY_VERSION" | sed -e 's/\.g.*//' -e 's/\.[^.]*$
 
 # Used in: .envrc
 
-export FISSILE_STEMCELL_VERSION=${FISSILE_STEMCELL_VERSION:-42.2-15.g38c573e-29.44}
+export FISSILE_STEMCELL_VERSION=${FISSILE_STEMCELL_VERSION:-42.2-15.g38c573e-29.48}
```

Comment:
Well, change looks good, and stemcell is properly pulled into vagrant.
The branch fails for `make diego-release`, and only that release.
I do not see that as blocker for merge.
I suspect an independent problem. (And maybe even with my checkout, despite doing `submodule sync && submodule update --init`).